torch/_tensor_docs.py

```diff
@@ -664,6 +664,20 @@ def add_docstr_all(method, docstr):
 In-place version of :meth:`~Tensor.bitwise_not`
 """)
 
+add_docstr_all('bitwise_invert',
+               r"""
+bitwise_invert() -> Tensor
+
+See :func:`torch.bitwise_invert`
+""")
+
+add_docstr_all('bitwise_invert_',
+               r"""
+bitwise_invert_() -> Tensor
+
+In-place version of :meth:`~Tensor.bitwise_invert`
```

Comment:
Should we also mention that this is an alias of `bitwise_not`. 

Cause from here user will go to `bitwise_invert` method and from there to `torch.bitwise_invert` to realize that it is an alias.

Reply:
Yeah but (see above) that's just how it is for all aliases at the moment. We could think about updating all of them in a separate PR